backend/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+
+  mysqldb:
+    image: mysql:8
+    container_name: mysqldb
+    ports:
+      - 3305:3306
+    restart: unless-stopped
+    volumes:
+      - db:/var/lib/mysql
+    environment:
+      MYSQL_DATABASE: 'SkyNest'
```

Comment:
usually practice is to use small letters not camel case

Reply:
Agreed, will change.